src/main/java/uk/gov/pay/connector/gateway/stripe/json/StripeSearchResponse.java

```
@@ -0,0 +1,28 @@
+package uk.gov.pay.connector.gateway.stripe.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StripeSearchResponse {
+
+    @JsonProperty("has_more")
+    private boolean hasMore;
+    @JsonProperty("data")
+    private List<StripePaymentIntent> paymentIntents;
+
+    public boolean hasMore() {
+        return hasMore;
+    }
+
+    public List<StripePaymentIntent> getPaymentIntents() {
+        return paymentIntents;
+    }
+
+    public Optional<StripePaymentIntent> getPaymentIntent() {
+        return paymentIntents != null ? paymentIntents.stream().findFirst() : Optional.empty();
```

Should we instead throw an exception and log an error to sentry if there is more than one payment intent returned? It should never happen, but would be a cause for concern